cpp/src/Slice/PythonUtil.cpp

```diff
@@ -1959,14 +1959,13 @@ Slice::Python::CodeVisitor::writeConstantValue(const TypePtr& type, const Syntax
             }
             case Slice::Builtin::KindString:
             {
-                string sv2 = toStringLiteral(value, "\a\b\f\n\r\t\v", "", Octal, 0);
-                string sv3 = toStringLiteral(value, "\a\b\f\n\r\t\v", "", UCN, 0);
+                const string controlChars = "\a\b\f\n\r\t\v";
+                const unsigned char cutOff = 0;
+                
+                string sv2 = toStringLiteral(value, controlChars, "", Octal, cutOff);
+                string sv3 = toStringLiteral(value, controlChars, "", UCN, cutOff);
 
-                _out << "\"" << sv2<< "\"";
-                if(sv2 != sv3)
-                {
-                    _out << " if _version_info_[0] < 3 else \"" << sv3 << "\"";
-                }
+                _out << "\"" << (sv2 == sv3 ? sv2 : sv3) << "\"";
```

Comment:
We only use `sv3` when they are different, when they are equal it doesn't matter, so can we just do

```
const string controlChars = "\a\b\f\n\r\t\v";
const unsigned char cutOff = 0;
_out << "\"" << toStringLiteral(value, controlChars, "", UCN, cutOff) << "\"";
```